docs/docs/server/openapi.yaml

```
@@ -17,6 +17,76 @@ servers:
 security:
   - CustomHeaderAuth: []
 paths:
+  /content/search:
+    post:
+      operationId: searchContent
+      summary: Execute a search query
```

```suggestion
      summary: Search content
```

nit: shorter, more to point